### DISCUSSION TITLE
창 여러개로 하면 밴먹나요?

### DISCUSSION THREAD

**Discussion by Zeta894** [changdoc/naver-vaccine-macro#108] [Q&A]
제목 그대로 한 병원의 신청 창을 여러 개 띄어놓고 하면 밴을 먹나요??

그리고 다른 병원 창을 각각 띄어놓고 해도 밴 먹나요??

방금 밴 한번 먹어서 소심해졌네요 ㅜ

**Comment by changdoc** [comment]
네 한시간동안 예약 신청 페이지에 접근이 차단됩니다.

**Reply by Zeta894** [reply]
아하.. 창 하나로 2초 주기 새로고침이면 새로고침 타이밍에 딱 나오길 기도해야겠네요
